packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -643,3 +745,136 @@ class _SnackBarState extends State<SnackBar> {
     );
   }
 }
+
+// Hand coded defaults based on Material Design 2.
+class _SnackbarDefaultsM2 extends SnackBarThemeData {
+  _SnackbarDefaultsM2(BuildContext context)
+      : _theme = Theme.of(context),
+        _colors = Theme.of(context).colorScheme;
+
+  final ThemeData _theme;
+  final ColorScheme _colors;
+
+  @override
+  double get elevation => 6.0;
```

Comment:
Given that this does not need to be computed, it could just be given in the constructor's super clause.

Reply:
Done